### ISSUE TITLE
mid3iconv: COMM frame not converted

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dhamaniasad/mutagen#128]
```
Example file: https://bugs.launchpad.net/ubuntu/+source/mutagen/+bug/331703
```

Original issue reported on code.google.com by `reiter.christoph@gmail.com` on 31 Oct 2012 at 3:56

**Comment by GoogleCodeExporter** [comment]
```
Issue 158 has been merged into this issue.
```

Original comment by `reiter.christoph@gmail.com` on 21 Aug 2013 at 6:25

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 4e177b147de3.
```

Original comment by `reiter.christoph@gmail.com` on 4 Sep 2013 at 4:46
- Changed state: **Fixed**
